VORP-Housing[Client-Server]/vorphousing_cl/vorphousing_cl_init.cs

```diff
@@ -63,25 +72,31 @@ private void SetHouseOwner(int houseId)
 
         public static async Task SetBlips()
         {
-            //249721687
-            foreach (var h in GetConfig.Config["Houses"])
+            try
             {
-                int _blip = Function.Call<int>((Hash)0x554D9D53F696D002, 1664425300, h["DoorsStatus"][0].ToObject<float>(), h["DoorsStatus"][1].ToObject<float>(), h["DoorsStatus"][2].ToObject<float>());
-                if (Houses.ContainsKey(h["Id"].ToObject<int>()))
+                //249721687
+                foreach (var h in GetConfig.Config["Houses"])
                 {
-                    if ((String.IsNullOrEmpty(Houses[h["Id"].ToObject<int>()].Identifier)))
-                    {
-                        Function.Call((Hash)0x74F74D3207ED525C, _blip, 249721687, 1);
-                    }
-                    else
+                    int _blip = Function.Call<int>((Hash)0x554D9D53F696D002, 1664425300, h["DoorsStatus"][0].ToObject<float>(), h["DoorsStatus"][1].ToObject<float>(), h["DoorsStatus"][2].ToObject<float>());
+                    if (Houses.ContainsKey(h["Id"].ToObject<int>()))
                     {
-                        Function.Call((Hash)0x74F74D3207ED525C, _blip, -2024635066, 1);
-                    }
-                    Function.Call((Hash)0x9CB1A1623062F402, _blip, h["Name"].ToString());
+                        if ((String.IsNullOrEmpty(Houses[h["Id"].ToObject<int>()].Identifier)))
```

Comment:
This can be improved to something like;
```cs
long blipSprite = -2024635066;

if ((String.IsNullOrEmpty(Houses[h["Id"].ToObject<int>()].Identifier)))
{
	blipSprite = 249721687;
}

Function.Call((Hash)0x74F74D3207ED525C, _blip, 249721687, 1);
```